apps/server/src/services/runtime-service/RuntimeService.ts

```diff
@@ -678,7 +678,8 @@ function broadcastResult(_target: any, _propertyKey: string, descriptor: Propert
     // we do the comparison by explicitly for each property
     // to apply custom logic for different datasets
 
-    const shouldForceTimerUpdate = getForceUpdate(RuntimeService.previousTimerUpdate, state.clock);
+    const shouldForceTimerUpdate =
+      state.timer.playback === Playback.Play && getForceUpdate(RuntimeService.previousTimerUpdate, state.clock);
```

Comment:
suggestion: I think the playback decision should be inside the getForceUpdate, so that this function is a complete encapsulation of the logic